Sources/Authgear.swift

```diff
@@ -95,6 +99,7 @@ public class Authgear: NSObject {
     private let workerQueue: DispatchQueue
 
     public private(set) var sessionState: SessionState = .unknown
+    public private(set) var authorizeRedirectionHandler: AuthorizeRedirectionHandler = { _ in }
```

Comment:
It's more conventional to make this optional, instead of having a default no-op value.